WordPress/src/main/java/org/wordpress/android/util/extensions/DialogExtensions.kt

```diff
@@ -39,15 +39,15 @@ fun BottomSheetDialog.fillScreen(isDraggable: Boolean = false) {
             MaterialR.id.design_bottom_sheet
         ) ?: return@setOnShowListener
 
+        bottomSheet.layoutParams?.let { layoutParams ->
+            layoutParams.height = WindowManager.LayoutParams.MATCH_PARENT
```

Comment:
Out of curiosity, I was wondering why we had to do this.

Reply:
If that's not set in landscape the Dialog won't fit the entire screen width, so I believe that's why this is done. But if that's done after setting the behavior it looks like it get's buggy (the dialog gets the desired width but it's not positioned properly, it's shifted to the right), so that's why I had to just change the order of the calls here.